specifyweb/frontend/js_src/lib/components/QueryBuilder/Header.tsx

```diff
@@ -88,17 +87,14 @@ export function QueryHeader({
               })}
         </H2>
         {!queryResource.isNew() && <QueryEditButton query={query} />}
-        <span className="ml-2 flex-1" />
         {!isScrolledTop && (
-          <Button.Small
+          <Button.Icon
+            icon="arrowCircleUp"
+            title="up"
```

Comment:
You forgot to localize "up". And "up" is not very description. How about "Scroll Up" or even better, "Scroll to Editor" (because up and down should be avoided as those are discriminating against the non-visual users)?

Reply:
title in line 322 in RecordSet.tsx should be changed too than?

Reply:
Ah, good catch! That is a bug. Can you please fix it on this branch?